app/models/post.rb

```diff
@@ -6,8 +6,8 @@ class Post < ApplicationRecord
   validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
 
```

Comment:
- [ ] Kindly make sure to use `CanCanCan` for Post authorization, you can add `load_and_authorize_resource` to your model

Reply:
Thanks for the comment but, I have it already in the posts_controller

Reply:
@amexabee, I am sorry for this, I checked the post model instead of the controller. in my opinion your project is completed, the next reviewer should approve your project